awx/main/conf.py

```diff
@@ -615,14 +615,15 @@ def _load_default_license_from_file():
     category=_('Jobs'),
     category_slug='jobs',
 )
+
 register(
     'MAX_FORKS',
     field_class=fields.IntegerField,
     allow_null=False,
-    default=0,
+    default=100,
```

Comment:
I heard various numbers thrown around in our chat earlier w/ @sivel and @jctanner.

Do we know if we have any users setting this higher in general?  Would 200 be on the safer side to avoid surprises for larger users?

Reply:
Maybe? Bumping it up to 200 sounds fine - Though if we take 25 forks per 4GB as our base line we'd probably want to know if `200` would prevent a bad scenario for folks just running the recommended `4GB` minimum.